docs/source/install/kernel_install.rst

```diff
@@ -53,3 +53,23 @@ For example, using conda environments:
 The ``--name`` value is used by Jupyter internally. These commands will overwrite
 any existing kernel with the same name. ``--display-name`` is what you see in
 the notebook menus.
+
+Using virtualenv or conda envs, you can make your IPython kernel in one env available to Jupyter in a different env. To do so, run ipykernel install from the kernel's env, with --prefix pointing to the Jupyter env:
```

Comment:
@minrk Rereading this, I'm inclined to leave as proposed in the PR. I'm guessing that the "why" is because conda populates the conda env name to the notebook interface, but I'm not seeing a real clean way to illustrate that concisely and without more confusion. Let's leave as proposed in the PR. One small grammar change below and this is good to merge or merge and iterate.

@pprzetacznik Thanks for the PR and improving the documentation! :sunny: